Makefile

```diff
@@ -68,20 +68,35 @@ SPECREVISION = $(shell echo `git describe --tags --dirty`)
 SPECREMARK = from git branch: $(shell echo `git symbolic-ref --short HEAD`) \
 	     commit: $(shell echo `git log -1 --format="%H"`)
 endif
-
-ATTRIBOPTS_NO_VERSION	= -a revdate="$(SPECDATE)" \
-			  -a revremark="$(SPECREMARK)" \
+# C++ for OpenCL doc revision scheme in aligned with its release date.
+# Revision naming scheme is as follows:
+# DOCREVYYYY.MM,
+# where YYYY corresponds to its release year,
+# MM corresponds to its release month.
+# Example for the release in Dec 2021 the revision is DOCREV2021.12.
+# Leave blank if the doc content does not correspond to any official revision.
+CXX4OPENCL_DOCREVISION = DOCREV2021.12
```

Comment:
As an alternative to leaving it blank, you could write `-unreleased` or `2021.12-next` perhaps?

Reply:
Yes, I would quite like to indicate that it is derived/on top of 2021.12, so adding `next` might be a good option
```suggestion
# Leave as 'DocRevYYYY.MM-Next' if the doc content does not correspond to any official revision.
# where DocRevYYYY.MM is the last released revision.
CXX4OPENCL_DOCREVISION = DocRev2021.12
```